cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/GlobalConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package io.smartcat.cassandra.diagnostics;
+
+import io.smartcat.cassandra.diagnostics.utils.Utils;
+
+/**
+ * Global configuration for Cassandra diagnostics.
+ */
+public class GlobalConfiguration {
+
+    /**
+     * System wide hostname. Set to override {@code InetAddress} querying.
+     */
+    public String hostname = Utils.resolveHostname();
+
+    /**
+     * System wide name. Set to differentiate between systems under observation.
+     */
+    public String systemName = "cassandra-cluster";
```

Comment:
I think that the default should be empty or null and then skipped in reporters/modules.